docs/design-docs/cli.md

```diff
@@ -0,0 +1,51 @@
+# Studio CLI
+
+## About this doc
+
+This document outlines the design and implementation details for the Studio CLI utility. It covers the high-level approach, data flows, and implementation details for this feature.
+
+## Context
+
+The Studio CLI (invoked with the `studio` command) is a globally available CLI utility allowing users to interact with various Studio features independently of the desktop application.
+
+## High level approach
+
+The CLI is independent of the main desktop app, but is written using mostly the same conventions. It's a node.js app written in Typescript, that's bundled into a single JS file using Webpack. Jest is used to test CLI modules in the same way as for regular Studio modules.
```

Comment:
Should we add a section to explain why we decided to "bundle" the files (the discussion we had previously)